### DISCUSSION TITLE
Some websites opening with weird layouts

### DISCUSSION THREAD

**Discussion by lypanov** [qutebrowser/qutebrowser#7788] [General] [answered]
I'm trying to migrate over to using qutebrowser as my primary browser, however frequently run into blockers due to websites loading, but with an obviously different style sheet / accessability mode enabled. Example URLs:
   https://www.ansible.com/resources/get-started
   https://console.cloud.google.com/welcome
Luckily I ran into the issue with ansible as without an account I doubt the Google Cloud Console URL issue is reproducible. I'm of course not certain these are even the same issue.. but seems to be a similar sort of behavior.

![image](https://github.com/qutebrowser/qutebrowser/assets/311/ed5b4d6e-6d0c-49e1-b2ad-fff3878360b2)

Hope someone can help!
Thanks!
Alex

**Comment by pylbrecht** [comment]
Could you provide some screenshots of the "weird layouts"?

Opening https://www.ansible.com/resources/get-started and https://console.cloud.google.com/welcome in qutebrowser looks fine layout-wise on my end.

My output of `:version`:
```
qutebrowser v2.5.4
Git commit: ee2c8c649 on master (2023-07-03 17:40:37 +0200)
Backend: QtWebEngine 6.5.1, based on Chromium 108.0.5359.220 (from api)
Qt: 6.5.1

CPython: 3.11.3
PyQt: 6.5.1

Qt wrapper: PyQt6 (via QUTE_QT_WRAPPER)

sip: 4.19.25
colorama: 0.4.6
jinja2: 3.1.2
pygments: 2.15.1
yaml: 6.0
adblock: 0.6.0
objc: no
PyQt6.QtWebEngineCore: 6.5.0
pdf.js: 3.7.107 (/usr/share/pdf.js/build/pdf.js)
sqlite: 3.42.0
QtNetwork SSL: OpenSSL 3.1.1 30 May 2023

Style: QFusionStyle
Platform plugin: xcb
OpenGL: Intel, 4.6 (Compatibility Profile) Mesa 23.1.3
Platform: Linux-6.4.3-arch1-2-x86_64-with-glibc2.37, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /usr/lib/python3.11/site-packages/qutebrowser
Using Python from /usr/bin/python3
Qt library executable path: /usr/lib/qt6, data path: /usr/share/qt6

Paths:
cache: /home/palbrecht/.cache/qutebrowser
config: /home/palbrecht/.config/qutebrowser
data: /home/palbrecht/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser
system data: /usr/share/qutebrowser

Autoconfig loaded: yes
Config.py: /home/palbrecht/.config/qutebrowser/config.py has been loaded
Uptime: 8:21:12
```

**Comment by The-Compiler** [comment, marked as answer]
It looks like those broke on Qt 5 - indeed works fine on Qt 6, see #7202.

**Comment by lypanov** [comment]
I've updated the original post to include the screenshot while running on Qt5 in case that helps anyone else seeing this behaviour.

**Comment by lypanov** [comment]
@The-Compiler Yup upgrading to Qt6 has fixed this. Hopefully distros will catch up soon. Thank you!
